scripts/config.py

```diff
@@ -18,6 +18,7 @@
 }
 SQL_FILE = os.path.join(parent, "scripts", "data.db")
 JSON_FILE = os.path.join(parent, "src", "static", "activities.json")
+SYNCED_FILE = os.path.join(parent, "imported.json")
```

Comment:
Where do you want to save `SYNCED_FILE`? Maybe use `FIT_FOLDER` instead of `parent`?